heartbeat/ZFS

```diff
@@ -83,7 +83,14 @@ END
 }
 
 zpool_is_imported () {
-	zpool list -H "$OCF_RESKEY_pool" > /dev/null
+	# Check if ZFS kstats exists
+    if [ -d  /proc/spl/kstat/zfs/ ] ; then
+		# Check the existence of kstats for the pool. If the stats exists, the pool was imported.
+        return `[ -d  /proc/spl/kstat/zfs/"${OCF_RESKEY_pool}" ]`
```

Comment:
I think it's better to not spawn a new process for this:
```
[ -d  /proc/spl/kstat/zfs/"${OCF_RESKEY_pool}" ]
return $?
```

Reply:
Ok. Maybe we can also omit `$?` in this case - we'll return the status of the last command anyway.

Reply:
Yes, it's a better idea.